model/charity.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const mongoose = require('mongoose');
```

Comment:
👍

Reply:
Some of the properites are abriviated "streedAdd" and should not be
It decreses readablity